app/common/patient/directives/patientHeader.html

```diff
@@ -1,26 +1,30 @@
-<div class="row bg-info-border bg-info content-border border-rouded-top">
+<div ng-class="{patientdeceased: vm.patient.dead}" class="row bg-info-border bg-info content-border border-rouded-top">
   <!--patient header -->
-  <div class="col-md-6">
-    <div class="header-text-lg">
-      <b>{{vm.patient.fullName}}</b>
+    <div class="col-md-6">
+      <div class="header-text-lg">
+        <b>{{vm.patient.fullName}}</b>
+      </div>
+      <div><!-- patient gender, age and phone number -->
+        <span>
+          <img src="../../images/{{vm.patient.gender === 'F' ? 'female.png' : 'male.png'}}"
+               class="img-responsive inline-align" width="24" height="24" alt="male">
+        </span>
+        <span>{{vm.patient.age.years}}</span>
+        <span>{{vm.patient.birthdateEstimated ? '~': ''}}({{vm.patient.birthdate | date:"MM/dd/yyyy"}})</span>
+        <!-- TODO: must check "Header Attribute Types" from openmrs configurations -->
+        <span ng-if="patient['Numero de Telefone']">Phone number: {{patient['Numero de Telefone']}}</span>
+      </div>
     </div>
-    <div><!-- patient gender, age and phone number -->
-      <span>
-        <img src="../../images/{{vm.patient.gender === 'F' ? 'female.png' : 'male.png'}}"
-             class="img-responsive inline-align" width="24" height="24" alt="male">
-      </span>
-      <span>{{vm.patient.age.years}}</span>
-      <span>{{vm.patient.birthdateEstimated ? '~': ''}}({{vm.patient.birthdate | date:"MM/dd/yyyy"}})</span>
-      <!-- TODO: must check "Header Attribute Types" from openmrs configurations -->
-      <span ng-if="patient['Numero de Telefone']">Phone number: {{patient['Numero de Telefone']}}</span>
+    <div class="col-md-6 pull-right go-to-right">
+      <ul class="list-unstyled">
+        <li ng-repeat="identifier in vm.patient.identifiers"
+            ng-class="{'header-text-lg' : identifier.preferred}">
+          {{identifier.identifierType.display}}: <b>{{identifier.identifier}}</b>
+        </li>
+      </ul>
     </div>
-  </div>
-  <div class="col-md-6 pull-right go-to-right">
-    <ul class="list-unstyled">
-      <li ng-repeat="identifier in vm.patient.identifiers"
-          ng-class="{'header-text-lg' : identifier.preferred}">
-        {{identifier.identifierType.display}}: <b>{{identifier.identifier}}</b>
-      </li>
-    </ul>
-  </div>
+
+    <p class="control-label"> {{'PATIENT_IS_DEAD' | translate}}
+      </br>{{'PATIENT_INFO_DEATH_DATE' | translate}}: {{vm.patient.deathDate | date: 'medium' }}
```

Comment:
Please use one of the date formats defined in: [constants.js](https://github.com/esaude/esaude-emr-poc/blob/master/app/common/application/constants.js). For ```short``` format we don't need to specify.